tests/test_unit/test_validation.py

```diff
@@ -34,6 +36,17 @@ def atlas_with_bad_reference_file():
     os.rename(bad_name, good_name)
 
 
+@pytest.fixture
+def atlas_with_missing_structure():
+    atlas = BrainGlobeAtlas("osten_mouse_100um")
+    modified_structures = atlas.structures.copy()
+    modified_structures.pop(688, None)
```

Comment:
```suggestion
    modified_structures.pop(688)
```
No need to specify a default here, I think.